config/cl-authorization/config.lisp

```diff
@@ -0,0 +1,205 @@
+;;;;;;;;;;;;;;;;;;;
+;;; delta messenger
+(in-package :delta-messenger)
+
+(setf *delta-handlers* nil)
+(add-delta-logger)
+(add-delta-messenger "http://deltanotifier/")
+
+;;;;;;;;;;;;;;;;;
+;;; configuration
+(in-package :client)
+(setf *log-sparql-query-roundtrip* t)
+(setf *backend* "http://virtuoso:8890/sparql"
+      ;; (list "http://triplestore:8890/sparql"
+      ;;       "http://triplestore1:8890/sparql"
+      ;;       "http://triplestore2:8890/sparql"
+      ;;       "http://triplestore3:8890/sparql"
+      ;;       )
+      )
+
+(in-package :server)
+(setf *log-incoming-requests-p* t)
+
+;;;;;;;;;;;;;;;;;
+;;; access rights
+
+(in-package :acl)
+
+(defparameter *access-specifications* nil
+  "All known ACCESS specifications.")
+
+(defparameter *graphs* nil
+  "All known GRAPH-SPECIFICATION instances.")
+
+(defparameter *rights* nil
+  "All known GRANT instances connecting ACCESS-SPECIFICATION to GRAPH.")
+
+(define-prefixes
+  :contacthub "http://data.lblod.info/vocabularies/contacthub/"
+  :besluit "http://data.vlaanderen.be/ns/besluit#"
+  :mandaat "http://data.vlaanderen.be/ns/mandaat#"
+  :persoon "http://data.vlaanderen.be/ns/persoon#"
+  :ext "http://mu.semte.ch/vocabularies/ext/"
+  :dct "http://purl.org/dc/terms/"
+  :adms "http://www.w3.org/ns/adms#"
+  :org "http://www.w3.org/ns/org#"
+  :person "http://www.w3.org/ns/person#"
+  :musession "http://mu.semte.ch/vocabularies/session/"
+  :foaf "http://xmlns.com/foaf/0.1/")
+
+(define-graph public ("http://mu.semte.ch/graphs/public")
+  ("ext:FileAddress" -> _)
+  ("http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#FileDataObject" -> _)
+  ("http://www.w3.org/ns/prov#Location" -> _)
+  ("besluit:Bestuurseenheid" -> _)
+  ("http://mu.semte.ch/vocabularies/ext/BestuurseenheidClassificatieCode" -> _)
```

Comment:
Might be usefull to use prefixes a much as possible to improve readability, e.g.
```
("ext:BestuurseenheidClassificatieCode" -> _)
```

Reply:
true i will update this to use the prefixes